packages/website/src/components/editor/LoadedEditor.tsx

```diff
@@ -232,6 +232,22 @@ export const LoadedEditor: React.FC<LoadedEditorProps> = ({
     resize();
   }, [resize, showAST]);
 
+  const domNode = sandboxInstance.editor.getContainerDomNode();
+  const resizeObserver = useMemo(() => {
+    return new ResizeObserver(() => {
+      resize();
+    });
+  }, []);
+
+  useEffect(() => {
+    if (domNode) {
+      resizeObserver.observe(domNode);
+
+      return (): void => resizeObserver.unobserve(domNode);
+    }
+    return (): void => {};
+  }, [domNode]);
```

Comment:
The `useMediaQuery.ts` file sparked me to wonder if this exists in `react-use` also... and it does! https://github.com/streamich/react-use/blob/master/docs/useMeasure.md

Reply:
IMO this hook seems helpful in general but a poor fit for this use case:

`useMeasure` returns the `width` of the `LoadedEditor` as 0. This is because the `LoadedEditor` isn't what contains the editor. The component that contains the editor is this one:
```
<div className={styles.tabCode}>
  <EditorEmbed />
</div>
```
But this doesn't have the `resize` function, so we need to pass it a prop for resize change in `Playground.tsx`, which we wanted to avoid.

The current code uses `ResizeObserver` to track `sandboxInstance.editor.getContainerDomNode()`, but `useMeasure` doesn't take any arguments to pass a separate DOM element in.

IMO keeping the resize logic for the editor contained in `LoadedEditor` unchanged is better than passing a prop around to use this hook. What do you think?

Reply:
Hmm, that does make sense - agreed. 😄